tests/acceptance/resources/keywords.resource

```diff
@@ -0,0 +1,88 @@
+*** Settings ***
+Library    SeleniumLibrary
+Library    OperatingSystem
+Library    Process
+Library    RequestsLibrary
+Library    Collections
+
+*** Keywords ***
+Start Rfhub2
+    [Arguments]    @{options}
+    [Documentation]
+    ...    Starts rfhub2 on the port given in the variable \${PORT}
+    ...    As a side effect this creates a suite variable named \${rfhub process},
+    ...    which is used by the 'Stop rfhub2' keyword.
+    ${python}    Evaluate    sys.executable    sys
+    ${rfhub2_process}    Start Process    ${python}    -m    rfhub2
+    # Sleep    2 seconds    # give the server a chance to start
+    Set Suite Variable    ${rfhub2_process}
+    Wait Until Keyword Succeeds    20 seconds    0.5 second
+    ...    Verify URL Is Reachable    /api/v1/health
+
+Stop Rfhub2
+    [Documentation]
+    ...    Stops the rfhub2 process created by "Start Rfhub2"
+    Terminate Process    ${rfhub2_process}
+    ${result}    Get Process Result
+    Run Keyword If    len('''${result.stderr}''') > 0
+    ...    Log    rfhub2 stderr: ${result.stderr}    DEBUG
+    
+Open Browser With App On Mainpage
+    Open Browser    url=localhost:8000    browser=gc
+    Maximize Browser Window
+
+Navigate To Main Page
+    Go To    localhost:8000
+
+Collections Count On Main Page Should Be ${n}
+    Reload Page
+    Element Text Should Be    ${main_page_collection_number}    Collections (${n})
+
+Start Cli Package With Options

```

Comment:
Maybe we could rename it to Run Rfhub2 Cli? We do not stop it later like with web server and it is more like executable than a separate package

Reply:
Roger!